spec.bs

```diff
@@ -218,10 +219,10 @@ In this section, we walk through some scenarios for Secure Payment Confirmation
 and the corresponding sample code for using this API. Note that these are
 example flows and do not limit the scope of how the API can be used.
 
-### Registration ### {#sctn-sample-registration}
+### Registration during a purchase ### {#sctn-sample-registration}
```

Comment:
Minor question: earlier in the spec we say "It is a goal of this specification to reduce authentication friction during checkout,"

The words "checkout" and "purchase" are both used in the specification. That is not a problem, but I'm wondering if there's a small benefit to referring to "Registration during a checkout"

Reply:
(I don't feel strongly about this.)

Reply:
SGTM, done.